cves/kernel/CVE-2012-6545.yml

```diff
@@ -462,7 +473,9 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: |
```

Comment:
Can you go into more detail on what managing memory looks like in code and/or how to do it on a higher level. You could also explain how mismanaging memory can affect a system.